server/mediapublic/mediapublic/models.py

```diff
@@ -259,8 +261,59 @@ def to_dict(self):
             secondary_website = self.secondary_website,
             creation_datetime = str(self.creation_datetime),
         )
-        return resp    
-    
+        return resp
+
+class PlaylistAssignments(Base, CreationMixin):
+    __tablename__ = 'playlist_assignments'
+    id = Column(Integer, primary_key=True)
+    playlist_id = Column(Integer, ForeignKey('playlists.id'))
+    recording_id = Column(Integer, ForeignKey('recordings.id'))
+
+    @classmethod
+    def get_by_playlist_id_and_recording_id(cls, pid, rid):
```

Comment:
I don't think this function is needed - see my comments below for views.py on where it is used.